lib/alephant/broker/cache/client.rb

```diff
@@ -39,7 +39,8 @@ def set(key, value, custom_ttl = nil)
           versioned_key = versioned(key)
           set_ttl       = custom_ttl || ttl
 
-          logger.info("#{self.class}#set - key: #{versioned_key}, value: #{value}, ttl: #{set_ttl}")
+          logger.debug("#{self.class}#set - key: #{versioned_key}, value: #{value}, ttl: #{set_ttl}")
```

Comment:
Wondering if we need both of these. Do we need to know which value has been added to the cache? Maybe, we should just keep the info one and remove the debug with value.

Reply:
@samfrench I agree, that would be the favorable option. I was being keen on other teams that expect the "value" to be logged?

Reply:
Happy to remove the `debug` log. 🙂

Reply:
@samfrench Just thinking through this. IMO, It might work better to retain both for debugging purposes and for users that prefer a more verbose log output? Also, removing it kind of beats the purpose of the work we did to add logging levels. 🙂

Reply:
One point would have been the duplicate log message, but you were also right about having the logging level in the logger, which this could have just been done...

At least with the level in the logger, we can now easily remove other logging messages, so that was good you didi it.